.github/workflows/pr_check_workflow.yml

```diff
@@ -14,16 +14,12 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [10.23.1]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
```

Comment:
This was the magic sauce to make it build? Nice.

Reply:
It was building without this, but I wanted to simplify. Getting all of the checks to pass still required admin intervention, and I documented the steps in [this comment](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/544#issuecomment-871094121).